app/models/game.rb

```diff
@@ -1,4 +1,63 @@
 class Game < ActiveRecord::Base
   belongs_to :users
   has_many :pieces
+  after_create :initialize_board!
+
+  def initialize_board!
```

Comment:
Would it be possible to take Joshua's suggestion
>If we add game: self to the attributes for creating pieces, we don't need to do pieces <<

one step further, and make this `def self.initialize_board!`? Would that save having to do it on every piece individually?